launcher.py

```diff
@@ -411,7 +436,8 @@ def copy_input(input_dir, working_dir, samples_file):
             until=[args.until] if args.until is not None else [],
             use_conda=True,
             conda_prefix=args.conda_prefix,
-            envvars=['CONDA_ENVS_PATH', 'CONDA_PKGS_DIRS']
+            envvars=['CONDA_ENVS_PATH', 'CONDA_PKGS_DIRS'],
+            keepgoing=True
```

Comment:
How does this relate to the current PR?

Reply:
It does not actually, it is related to the GRAPE-118. Pipeline needs to keep going after some batches fall.

Reply:
Pls move this change to `GRAPE-118`.

Reply:
Ok, done.